Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["ctru-rs", "ctru-sys", "ctru-sys/docstring-to-rustdoc"]
+members = ["ctru-rs", "ctru-sys", "ctru-sys/bindgen-ctru-sys"]
```

Comment:
you might need to set `default-members = ["ctru-rs", "ctru-sys"]` to avoid trying to build the bindgen package for 3DS as part of CI, I think I had to do that in `citro3d` so that `cargo 3ds check --workspace` still worked.